analytic_engine/src/compaction/picker.rs

```diff
@@ -420,8 +477,17 @@ impl SizeTieredPicker {
 
 /// Time window compaction strategy
 /// See https://github.com/jeffjirsa/twcs/blob/master/src/main/java/com/jeffjirsa/cassandra/db/compaction/TimeWindowCompactionStrategy.java
-#[derive(Default)]
-pub struct TimeWindowPicker {}
+pub struct TimeWindowPicker {
+    pick_by_seq: bool,
+}
```

Comment:
Define the trait, and refactor like this?
```
struct TimeWindowPicker {
    new_bucket_picker: Box<dyn Picker>,
    old_bucket_picker: Box<dyn Picker>,
}
```

Reply:
Not necessary, old implementation will be removed when new is stable.